synapse/config/workers.py

```diff
@@ -88,7 +88,7 @@ class Config:
         allow_mutation = False
 
 
-class InstanceLocationConfig(ConfigModel):
+class InstanceTCPLocationConfig(ConfigModel):
```

Comment:
For TCP, we seem to do it both ways so my two cents is `Tcp` but don't really care beyond aligning.

https://github.com/matrix-org/synapse/blob/ba97b39881e296f4775b8f6dd18edb98a3dc733f/synapse/replication/tcp/redis.py#L313

https://github.com/matrix-org/synapse/blob/ba97b39881e296f4775b8f6dd18edb98a3dc733f/synapse/config/server.py#L217